requirements.txt

```diff
@@ -5,3 +5,5 @@ numpy>=1.13
 urllib3>=1.21.1
 python-dateutil>=2.8.0
 websocket-client>=1.0.1
+websockets>=10.0; python_version >= '3.7'
+websockets>=9.1; python_version < '3.7'
```

Comment:
I think we no longer support Python 3.6. `websockets>=9.1; python_version < '3.7'` is no longer needed. 

Couldn't find in the copied PR why it was needed anyway...

Reply:
Agree this can be removed